### PR TITLE
first about us commit

### PR DESCRIPTION
**Pull request by JasonWeakley**
I created the about-us.html and about-us.css files and wrote the code for the HTML file, the css has not been done yet.

### CODE REVIEW

**Comment by shupac800** [comment]
:+1:

**Comment by shupac800** [comment]
:+1:

File: about-us.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      <h3>Our History</h3>
+      <p>Located in Diagon Alley, London, The Confundus Charm was founded in 1132 A.D. to equip witches and wizards, and generations of Hogwarts students, with magical items of every sort.</p>
+      <p>We specialize in wizarding objects as well as ancient tomes and modern texts for spell casting and potion making.</p>
+      <p>Our namesake and most popular charm has helped us keep our doors open to loyal customers throughout the centuries. The Confundus Charm has been open seven days a week, every day of the year, since we first opened our doors.</p>
+      <p>Shop keepers have been instrumental in many events throughout London's history including using the confundus charm to thwart the Gunpowder Treason Plot, and confusing German bombers during the Battle of Britain.</p>
+      <p>During the years of Lord Voldemort's acts of terror a confundus charm was placed around the shop's entrance to keep all but our most loyal customers from entering the premises.</p>
+    </section>
+    <section>
+      <div>
```
Comment (craftylildev) [inline about-us.html]: i would recommend putting a class on the div element so you can style the img & span tags easier

**Comment by craftylildev** [comment]
Hey I found one thing to review, as soon as that's done - you're good to go.

File: about-us.html (hunk 1 above)
Comment (JasonWeakley) [inline about-us.html]: Cool thanks, I will do that for the next commit

**Comment by craftylildev** [comment]
:+1:

**Pull request merged**